packages/@webex/internal-plugin-metrics/src/metrics.js

```diff
@@ -70,7 +70,6 @@ const Metrics = WebexPlugin.extend({
       domain:
         typeof window !== 'undefined' ? window.location.hostname || 'non-browser' : 'non-browser', // Check what else we could measure
       client_id: this.webex.credentials.config.client_id,
-      user_id: this.webex.internal.device.userId,
```

Comment:
@arun3528, @meta-dipanshu-sharma is removing the user_id tags as part of this PR as he removed '@webex/internal-plugin-device', I guess this will be ok, what do you think ??

Reply:
ya should not be a issue